Test/Exam1/exercice2/src/exercice2/SortedContainer.java

```diff
@@ -0,0 +1,45 @@
+package exercice2;
+
+public class SortedContainer {
+	int[] elements = new int[5000];
+	private int size;
+	
+	// Cr�ation de l'objet
+	public SortedContainer(int[] list) {
+		sort(list, list.length);
+		size = list.length;
+	}
+	
+	// Ajout d'un �l�ment
+	public void add(int value) {
+		int i = size;
+		elements[i] = value;
+		sort(elements, elements.length);
+	}
+	
+	public int[] elements() {
+		return elements;	
```

Comment:
This return **all** the items of elements.
What we need is the effective items (items from 0 to size-1)